models/task/task.go

```diff
@@ -25,8 +26,8 @@ type TaskStruct struct {
 	database    db.DB
 }
 
-func NewTask(id int64, listID int64, userID int64, issueNumber sql.NullInt64, title string, description string) *TaskStruct {
-	task := &TaskStruct{Id: id, ListId: listID, UserId: userID, IssueNumber: issueNumber, Title: title, Description: description}
+func NewTask(id int64, listID int64, projectID int64, userID int64, issueNumber sql.NullInt64, title string, description string) *TaskStruct {
```

Comment:
__[golint]__
* [golint] exported function NewTask should have comment or be unexported